README.md

```diff
@@ -15,6 +15,7 @@
 |local_code_path  |False | the path of local code diretory of apisix or dashboard, which depends on the app parameter|local_code_path=/home/vagrant/apisix|
 |image_base|False |the environment for packaging, if type is `rpm` the default image_base is `centos`, if type is `deb` the default image_base is `ubuntu`|image_base=centos|
 |image_tag|False |the environment for packaging, it's value can be `16.04\|18.04\|20.04\|6\|7\|8`, if type is `rpm` the default image_tag is `7`, if type is `deb` the default image_tag is `20.04`|image_tag=7|
+|buildx|False|if `true`, use buildx to build docker images, which may speed up GitHub Actions|buildx=true|
```

Comment:
```suggestion
|buildx|false|if `true`, use buildx to build docker images, which may speed up GitHub Actions|buildx=true|
```

Reply:
I'm confused. this uppercase is aligned to the above

Reply:
@qsliu2017 if we use uppercase, how about changing the `true` to `True` alongside? 😄

Reply:
@qsliu2017 Hi, any update? 😄

Reply:
@qsliu2017 sorry for my mistake. It's OK here.